logos/src/logos/dbutils/dbmanager.py

```
@@ -0,0 +1,845 @@
+        pk = self.insert("models", {"name": name, "endpoint": endpoint})
+        return {"result": f"Created Model", "model_id": pk}, 200
+
+    def add_service(self, logos_key: str, name: str):
+        if not self.check_authorization(logos_key):
+            return {"error": "Database changes only allowed for root user."}, 500
+        pk = self.insert("services", {"name": name})
+        api_key = generate_logos_api_key(name)
+        ppk = self.insert("process", {"logos_key": api_key, "service_id": pk, "name": name})
+        return {"result": f"Created Service.", "service-id": pk, "process-id": ppk, "logos-key": api_key}, 200
+
+    def add_token_type(self, name: str, description: str = "", exist_ok = True):
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)